google-cloud-pubsub/pom.xml

```diff
@@ -72,6 +102,12 @@
       <version>3.4</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-all</artifactId>
+        <version>1.9.5</version>
+        <scope>test</scope>
+    </dependency>
```

Comment:
This is way too big to do right now: https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1596